configs/workflows-mt.yaml

```diff
@@ -0,0 +1,377 @@
+Workflows:
+  - Name: Sequencing Noninterleaved
+    Collection: omics_processing_set
+    Enabled: True
+    Filter Output Objects:
+    - Metagenome Raw Read 1
+    - Metagenome Raw Read 2
+
+  - Name: Sequencing Interleaved
+    Collection: omics_processing_set
+    Enabled: True
+    Filter Output Objects:
+    - Metagenome Raw Reads
+
+  - Name: Sequencing
+    Type: nmdc:MetagenomeSequencingActivity
+    Enabled: False
+    Git_repo: https://github.com/microbiomedata/RawSequencingData
+    Version: v1.0.0
+    Collection: metagenome_sequencing_activity_set
+    Outputs: []
+
+  - Name: Reads QC
```

Comment:
We should change the name to be metaT specific for both of the Reads QC workflows.

Reply:
we left this as is to minimize schema changes/migrations, we can revisit after berkeley refactor rollout.

Reply:
Done